source/edge-server/connect.txt

```diff
@@ -63,12 +63,49 @@ documentation for your preferred SDK:
 - React Native SDK: :ref:`react-native-connect-to-mongodb-realm-backend-app`
 - Swift SDK: :ref:`ios-init-appclient`
 
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server currently supports a subset of authentication providers when
+connecting with an SDK:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`api-key-authentication`
+- :ref:`custom-function-authentication`
+
+.. _edge-server-device-sdk-roaming:
+
+Roaming Between Edge Servers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SDK clients can roam between Edge Server instances, or between an Edge Server
+instance and Atlas.
```

Comment:
Do we usually say Device SDK clients connect to Atlas? Or do we say they connect to app services?

Reply:
We're inconsistent, but I have generally been hand-waving this and saying "Atlas" since we added the ability to create a Device Sync service in the Data Services tab.